packages/user/src/supertokens/index.ts

```diff
@@ -12,7 +13,11 @@ const superTokens = (config: AppConfig) => {
       apiDomain: config.apiBaseUrl,
       apiBasePath: config.authBasePath || SUPERTOKENS_API_BASE_PATH_DEFAULT,
     },
-    recipeList: [Session.init(), ThirdPartyEmailPassword.init()],
+    recipeList: [
+      Session.init(),
+      EmailVerification.init(),
+      ThirdPartyEmailPassword.init(),
+    ],
```

Comment:
Do we need to check if feature is enabled to add EmailVerification in recipeList?